src/scanpy/preprocessing/_scrublet/core.py

```diff
@@ -28,13 +27,7 @@
 __all__ = ["Scrublet"]
 
 
-if sys.version_info > (3, 10):
```

Comment:
this check was wrong, should have been `>=`, all the `kw_only` features were added in 3.10: https://docs.python.org/3/library/dataclasses.html#dataclasses.KW_ONLY